osu/objects/user.py

```diff
@@ -547,3 +575,6 @@ def __init__(self, data, attr_type):
             print(f"WARNING: Unrecognized attr_type for CurrentUserAttributes: \"{attr_type}\"")
             for k, v in data.items():
                 setattr(self, k, v)
+
+    def __repr__(self):
```

Comment:
I think it would be better to only give the type attribute to prettify.

Reply:
solved in f227b2e034393567de3891b7351c8f3ab3462b35